main.cpp

```diff
@@ -3,70 +3,68 @@
 #include <memory>
 #include <string>
 
-
 namespace {
+  // The UI abstraction is contained here. Subclasses implement the look and feel.
```

Comment:
Hi Sassah, as discussed offline, please revert the indentation added. Thanks!

Reply:
ok